src/app/cli/cli.ts

```diff
@@ -192,10 +212,25 @@ export async function startHeadlessCli(agent: SaikiAgent, prompt: string): Promi
     // Common initialization
     await _initCli(agent);
     try {
-        // Execute the task
-        // reset conversation for headless mode
-        await agent.resetConversation();
-        await agent.run(prompt);
+        // Check if this is a slash command
+        const parsed = parseInput(prompt);
+
+        if (parsed.type === 'command') {
+            // Execute slash command
+            if (!parsed.command) {
+                console.log(
+                    chalk.yellow('💡 No command specified. Use /help to see available commands')
+                );
+                return;
+            }
+
+            await executeCommand(parsed.command, parsed.args || [], agent);
+        } else {
+            // Execute the task as a regular AI prompt
+            // reset conversation for headless mode
+            await agent.resetConversation();
```

Comment:
i've been thinking we could remove this conversation reset as well and could let the headless CLI continue the previous conversation

can decide this separately independent of this PR but we can think about the UX we want for the headless version